packages/app-store/stripepayment/lib/customer.ts

```diff
@@ -82,3 +82,28 @@ export async function deleteStripeCustomer(user: UserType): Promise<string | nul
 
   return deletedCustomer.id;
 }
+
+export async function retrieveOrCreateStripeCustomerByEmail(email: string, stripeAccountId: string) {
+  console.log("🚀 ~ file: customer.ts:87 ~ retrieveOrCreateStripeCustomerByEmail ~ email:", email);
+  const customer = await stripe.customers.list(
+    {
+      email,
+      limit: 1,
+    },
+    {
+      stripeAccount: stripeAccountId,
+    }
+  );
```

Comment:
This customer data isn't crucial like our team's customers as we were never collecting this data before. We need a customer to attach a payment method to